actions/workflows/st2workroom_st2enterprise_test.yaml

```diff
@@ -6,6 +6,12 @@
     st2_password: "cibuild"
   chain:
     -
+      name: "destroy_vm_if_exists"
```

Comment:
This is an interesting choice. I can live with it since it is our primary operational pattern.